tests/helpers.go

```diff
@@ -1,16 +1,20 @@
 package tests
 
 import (
-	"math/rand"
+	"encoding/binary"
+	"math/rand/v2"
 	"testing"
 
 	"github.com/nspcc-dev/neo-go/pkg/neotest"
 	"github.com/nspcc-dev/neo-go/pkg/vm/stackitem"
 )
 
 func randomBytes(n int) []byte {
+	if n < 8 {
+		panic("too small request")
```

Comment:
unexpected restriction, comment at least?

Reply:
You'll see it immediately if you're to hit it, then you can write a bit more fancy code to deal with it.